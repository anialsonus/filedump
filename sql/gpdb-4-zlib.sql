-- create table <name> (c1 int, c2 text)
--   with (appendonly=true, compresstype=zlib)
\! pg_filedump -z -M -k -T zlib test/ao.zlib.gp4

-- create table <name> (c1 int, c2 text)
--   with (appendonly=true, orientation=column, compresstype=zlib)
\! pg_filedump -z -M -k -O column -T zlib test/co.zlib.gp4
