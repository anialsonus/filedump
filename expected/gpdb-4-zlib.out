-- create table <name> (c1 int, c2 text)
--   with (appendonly=true, compresstype=zlib)
\! pg_filedump -z -M -k -T zlib test/ao.zlib.gp4

*******************************************************************
* PostgreSQL File/Block Formatted Dump Utility
*
* File: test/ao.zlib.gp4
* Options used: -z -M -k -T zlib
*******************************************************************

 Initial Bytes Read = 528

***** Block    0 Start ***( Position:    0) *************************
<AO Header> -----
 HeaderKind: 1 
 Header Checksum Valid (checksum 0x5db87473)
(AoHeaderKind_SmallContent)
 executorBlockKind: 1 rowCount: 31
 headerLen: 24 uncompressedLen: 566 compressedLen: 238
 firstRowNumber: 1
Content checksum valid (checksum: 0xa69d2375)

***** Block    0 End (Total Block length:  264) ******************
 Initial Bytes Read = 264

***** Block    1 Start ***( Position:  264) *************************
<AO Header> -----
 HeaderKind: 1 
 Header Checksum Valid (checksum 0x99d89310)
(AoHeaderKind_SmallContent)
 executorBlockKind: 1 rowCount: 31
 headerLen: 24 uncompressedLen: 566 compressedLen: 238
 firstRowNumber: 101
Content checksum valid (checksum: 0xc70c8186)

***** Block    1 End (Total Block length:  264) ******************
 Initial Bytes Read = 0

 Done DumpAppendOnlyFileContents.....
-- create table <name> (c1 int, c2 text)
--   with (appendonly=true, orientation=column, compresstype=zlib)
\! pg_filedump -z -M -k -O column -T zlib test/co.zlib.gp4

*******************************************************************
* PostgreSQL File/Block Formatted Dump Utility
*
* File: test/co.zlib.gp4
* Options used: -z -M -k -O column -T zlib
*******************************************************************

 Initial Bytes Read = 208

***** Block    0 Start ***( Position:    0) *************************
<AO Header> -----
 HeaderKind: 1 
 Header Checksum Valid (checksum 0x2963ec09)
(AoHeaderKind_SmallContent)
 executorBlockKind: 1 rowCount: 31
 headerLen: 24 uncompressedLen: 140 compressedLen: 74
 firstRowNumber: 1
Content checksum valid (checksum: 0x85e2e077)

	<AO Content Info> -----
	 Decompressing using "ZLIB" with compressionLevel: 0
	 DatumStreamVersion_Original  flags: 0
	 ndatum: 31 nullSize: 0 Size: 124
***** Block    0 End (Total Block length:  104) ******************
 Initial Bytes Read = 104

***** Block    1 Start ***( Position:  104) *************************
<AO Header> -----
 HeaderKind: 1 
 Header Checksum Valid (checksum 0x75a7f00b)
(AoHeaderKind_SmallContent)
 executorBlockKind: 1 rowCount: 31
 headerLen: 24 uncompressedLen: 140 compressedLen: 74
 firstRowNumber: 101
Content checksum valid (checksum: 0xbb93f16a)

	<AO Content Info> -----
	 Decompressing using "ZLIB" with compressionLevel: 0
	 DatumStreamVersion_Original  flags: 0
	 ndatum: 31 nullSize: 0 Size: 124
***** Block    1 End (Total Block length:  104) ******************
 Initial Bytes Read = 0

 Done DumpAppendOnlyFileContents.....
